source/common/CMakeLists.txt

```diff
@@ -23,8 +23,8 @@ if (NOT DEFINED UMF_REPO)
 endif()
 
 if (NOT DEFINED UMF_TAG)
-    # v0.9.x 19.08.2024: Merge pull request #688 ...
-    set(UMF_TAG 59c4150b7120a7af5b3c8eb2d9b8bbb5d2e96aa3)
+    # v0.9.x 12.09.2024: 0.9.0 release
+    set(UMF_TAG 57e819999a262b79bcceec47cca24f5015611ea1)
```

Comment:
Do you have a tag we could use here instead of the commit hash?

Reply:
yes, of course we have a tag... but we tend to use commits, as tags can be force-pushed, and not all forks/clones contain tags - if you prefer to use a tag I'll do that :)

Reply:
I think for the release branches using a release tag like [v0.9.0](https://github.com/oneapi-src/unified-memory-framework/releases/tag/v0.9.0) makes it much clearer to see the tag is pointing to the correct place.

I'd hope that release tags aren't getting changed after creation...

Reply:
Yeah, we don't want to force-push tags, but it still possible.

Changed to tag.